atheneum/atheneum.py

```diff
@@ -1,3 +1,17 @@
-class Atheneum:
-    def answer(self):
+"""A boilerplate module for Python packate/library design"""
+
+class Atheneum: #pylint:disable=too-few-public-methods
+    """The atheneum class has all the answers"""
+
+    def __init__(self) -> None:
+        pass
+
+    def answer(self, exclamation_str):
+        """
+        Return the answer to Life, the Universe, and Everything.
+
+        :param exclamation_str:     Print how you feel about the answer
+        """
```

Comment:
At some point we'll want to standardize how we do API documentation. I assume this is a standard syntax for something like sphynx?

Reply:
Yeah, this is ReStructured Text that sphinx uses. Tthis looks like a pointless function signature, but I wanted to contrive an example to use the RST comments...